### ISSUE TITLE
【EESS】New submissions for Wed, 11 Oct 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#733]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imaging
There is no result 
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
## Keyword: mobile
There is no result 
### Adversarial Masked Image Inpainting for Robust Detection of Mpox and  Non-Mpox
 - **Authors:** Yubiao Yue, Zhenzhang Li
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2310.06318
 - **Pdf link:** https://arxiv.org/pdf/2310.06318
 - **Abstract**
 Due to the lack of efficient mpox diagnostic technology, mpox cases continue to increase. Recently, the great potential of deep learning models in detecting mpox and non-mpox has been proven. However, existing models learn image representations via image classification, which results in they may be easily susceptible to interference from real-world noise, require diverse non-mpox images, and fail to detect abnormal input. These drawbacks make classification models inapplicable in real-world settings. To address these challenges, we propose "Mask, Inpainting, and Measure" (MIM). In MIM's pipeline, a generative adversarial network only learns mpox image representations by inpainting the masked mpox images. Then, MIM determines whether the input belongs to mpox by measuring the similarity between the inpainted image and the original image. The underlying intuition is that since MIM solely models mpox images, it struggles to accurately inpaint non-mpox images in real-world settings. Without utilizing any non-mpox images, MIM cleverly detects mpox and non-mpox and can handle abnormal inputs. We used the recognized mpox dataset (MSLD) and images of eighteen non-mpox skin diseases to verify the effectiveness and robustness of MIM. Experimental results show that the average AUROC of MIM achieves 0.8237. In addition, we demonstrated the drawbacks of classification models and buttressed the potential of MIM through clinical validation. Finally, we developed an online smartphone app to provide free testing to the public in affected areas. This work first employs generative models to improve mpox detection and provides new insights into binary decision-making tasks in medical images.
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Empirical Evaluation of the Segment Anything Model (SAM) for Brain Tumor  Segmentation
 - **Authors:** Mohammad Peivandi, Jason Zhang, Michael Lu, Dongxiao Zhu, Zhifeng Kou
 - **Subjects:** Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2310.06162
 - **Pdf link:** https://arxiv.org/pdf/2310.06162
 - **Abstract**
 Brain tumor segmentation presents a formidable challenge in the field of Medical Image Segmentation. While deep-learning models have been useful, human expert segmentation remains the most accurate method. The recently released Segment Anything Model (SAM) has opened up the opportunity to apply foundation models to this difficult task. However, SAM was primarily trained on diverse natural images. This makes applying SAM to biomedical segmentation, such as brain tumors with less defined boundaries, challenging. In this paper, we enhanced SAM's mask decoder using transfer learning with the Decathlon brain tumor dataset. We developed three methods to encapsulate the four-dimensional data into three dimensions for SAM. An on-the-fly data augmentation approach has been used with a combination of rotations and elastic deformations to increase the size of the training dataset. Two key metrics: the Dice Similarity Coefficient (DSC) and the Hausdorff Distance 95th Percentile (HD95), have been applied to assess the performance of our segmentation models. These metrics provided valuable insights into the quality of the segmentation results. In our evaluation, we compared this improved model to two benchmarks: the pretrained SAM and the widely used model, nnUNetv2. We find that the improved SAM shows considerable improvement over the pretrained SAM, while nnUNetv2 outperformed the improved SAM in terms of overall segmentation accuracy. Nevertheless, the improved SAM demonstrated slightly more consistent results than nnUNetv2, especially on challenging cases that can lead to larger Hausdorff distances. In the future, more advanced techniques can be applied in order to further improve the performance of SAM on brain tumor segmentation.
### Compression Ratio Learning and Semantic Communications for Video Imaging
 - **Authors:** Bowen Zhang, Zhijin Qin, Geoffrey Ye Li
 - **Subjects:** Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2310.06246
 - **Pdf link:** https://arxiv.org/pdf/2310.06246
 - **Abstract**
 Camera sensors have been widely used in intelligent robotic systems. Developing camera sensors with high sensing efficiency has always been important to reduce the power, memory, and other related resources. Inspired by recent success on programmable sensors and deep optic methods, we design a novel video compressed sensing system with spatially-variant compression ratios, which achieves higher imaging quality than the existing snapshot compressed imaging methods with the same sensing costs. In this article, we also investigate the data transmission methods for programmable sensors, where the performance of communication systems is evaluated by the reconstructed images or videos rather than the transmission of sensor data itself. Usually, different reconstruction algorithms are designed for applications in high dynamic range imaging, video compressive sensing, or motion debluring. This task-aware property inspires a semantic communication framework for programmable sensors. In this work, a policy-gradient based reinforcement learning method is introduced to achieve the explicit trade-off between the compression (or transmission) rate and the image distortion. Numerical results show the superiority of the proposed methods over existing baselines.
### Streaming Probabilistic PCA for Missing Data with Heteroscedastic Noise
 - **Authors:** Kyle Gilman, David Hong, Jeffrey A. Fessler, Laura Balzano
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2310.06277
 - **Pdf link:** https://arxiv.org/pdf/2310.06277
 - **Abstract**
 Streaming principal component analysis (PCA) is an integral tool in large-scale machine learning for rapidly estimating low-dimensional subspaces of very high dimensional and high arrival-rate data with missing entries and corrupting noise. However, modern trends increasingly combine data from a variety of sources, meaning they may exhibit heterogeneous quality across samples. Since standard streaming PCA algorithms do not account for non-uniform noise, their subspace estimates can quickly degrade. On the other hand, the recently proposed Heteroscedastic Probabilistic PCA Technique (HePPCAT) addresses this heterogeneity, but it was not designed to handle missing entries and streaming data, nor does it adapt to non-stationary behavior in time series data. This paper proposes the Streaming HeteroscedASTic Algorithm for PCA (SHASTA-PCA) to bridge this divide. SHASTA-PCA employs a stochastic alternating expectation maximization approach that jointly learns the low-rank latent factors and the unknown noise variances from streaming data that may have missing entries and heteroscedastic noise, all while maintaining a low memory and computational footprint. Numerical experiments validate the superior subspace estimation of our method compared to state-of-the-art streaming PCA algorithms in the heteroscedastic setting. Finally, we illustrate SHASTA-PCA applied to highly-heterogeneous real data from astronomy.
### On the Peak AoI of UAV-assisted IoT Networks: A Stochastic Geometry  Approach
 - **Authors:** Yujie Qin, Mustafa A. Kishk, Mohamed-Slim Alouini
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2310.06366
 - **Pdf link:** https://arxiv.org/pdf/2310.06366
 - **Abstract**
 In this paper, we analyze the peak age of information (PAoI) in UAV-assisted internet of thing (IoT) networks, in which the locations of IoT devices are modeled by a Mat\'{e}rn cluster process (MCP) and UAVs are deployed at the cluster centers to collect the status updates from the devices. Specifically, we consider that IoT devices can either monitor the same physical process or different physical processes and UAVs split their resources, time or bandwidth, to serve the devices to avoid inter-cluster interference. Using tools from stochastic geometry, we are able to compute the mean activity probability of IoT devices and the conditional success probability of an individual device. We then use tools from queuing theory to compute the PAoI under two load models and two scenarios for devices, respectively. Our numerical results show interesting system insights. We first show that for a low data arrival rate, increasing the number of correlated devices can improve the PAoI for both load models. Next, we show that even though the time-splitting technique causes higher interference, it has a limited impact on the mean PAoI, and the mean PAoI benefits more from the time-splitting technique. This is because of the nature of UAV communication, especially at places where devices (users) are spatially-clustered: shorter transmission distances and better communication channels, comparing the links established by the cluster UAV and serving devices (users) to links established by interferers.
### 6G Wireless Communications in 7-24 GHz Band: Opportunities, Techniques,  and Challenges
 - **Authors:** Zhuangzhuang Cui, Peize Zhang, Sofie Pollin
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2310.06425
 - **Pdf link:** https://arxiv.org/pdf/2310.06425
 - **Abstract**
 The sixth generation (6G) wireless communication nowadays is seeking a new spectrum to inherit the pros and discard the cons of sub-6 GHz, millimeter-wave (mmWave), and sub-terahertz (THz) bands. To this end, an upper mid-band, Frequency Range (FR) spanning from 7 GHz to 24 GHz, also known as FR3, has emerged as a focal point in 6G communications. Thus, as an inexorable prerequisite, a comprehensive investigation encompassing spectrum utilization and channel modeling is the first step to exploit potential applications and future prospects of using this FR in the 6G ecosystem. In this article, we provide FR3 deployment insights into emerging technologies including non-terrestrial network (NTN), massive multi-input multi-output (mMIMO), reconfigurable intelligent surface (RIS), and joint communications and sensing (JCAS). Furthermore, leveraging ray-tracing simulations, our investigation unveils the channel characteristics in FR3 are close to those in the sub-6 GHz band. The analysis of RIS-aided communication shows a higher spectral efficiency achieved in FR3 compared to other FRs when using the same RIS size. Finally, challenges and promising directions are discussed for FR3-based communication systems.
### Small-Signal Stability and SCR Enhancement of Offshore WPPs with  Synchronous Condensers
 - **Authors:** Sulav Ghimire, Kanakesh V. Kkuni, Emerson D. Guest, Kim H. Jensen, Guangya Yang
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2310.06457
 - **Pdf link:** https://arxiv.org/pdf/2310.06457
 - **Abstract**
 Synchronous condensers (SCs) have been reported to improve the overall stability and short-circuit power of a power system. SCs are also being integrated into offshore wind power plants (WPPs) for the same reason. This paper, investigates the effect of synchronous condensers on an offshore wind power plant with grid-following (GFL) and grid-forming (GFM) converter controls. Primarily, the effect of synchronous condensers can be two-fold: (1) overall stability enhancement of the WPP by providing reactive power support, (2) contribution to the effective short circuit ratio (SCR) of the WPP by fault current support. Therefore, this paper focuses on studies concerning these effects on an aggregated model of a WPP connected to the grid. To that end, a state-space model of the test system is developed for small-signal stability assessment and the synchronous condenser's effect on its stability. In addition, a mathematical explanation of SCR enhancement with synchronous condenser is provided and is verified with time-domain electromagnetic transient simulations.
### Deep Cardiac MRI Reconstruction with ADMM
 - **Authors:** George Yiasemis, Nikita Moriakov, Jan-Jakob Sonke, Jonas Teuwen
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Medical Physics (physics.med-ph)
 - **Arxiv link:** https://arxiv.org/abs/2310.06628
 - **Pdf link:** https://arxiv.org/pdf/2310.06628
 - **Abstract**
 Cardiac magnetic resonance imaging is a valuable non-invasive tool for identifying cardiovascular diseases. For instance, Cine MRI is the benchmark modality for assessing the cardiac function and anatomy. On the other hand, multi-contrast (T1 and T2) mapping has the potential to assess pathologies and abnormalities in the myocardium and interstitium. However, voluntary breath-holding and often arrhythmia, in combination with MRI's slow imaging speed, can lead to motion artifacts, hindering real-time acquisition image quality. Although performing accelerated acquisitions can facilitate dynamic imaging, it induces aliasing, causing low reconstructed image quality in Cine MRI and inaccurate T1 and T2 mapping estimation. In this work, inspired by related work in accelerated MRI reconstruction, we present a deep learning (DL)-based method for accelerated cine and multi-contrast reconstruction in the context of dynamic cardiac imaging. We formulate the reconstruction problem as a least squares regularized optimization task, and employ vSHARP, a state-of-the-art DL-based inverse problem solver, which incorporates half-quadratic variable splitting and the alternating direction method of multipliers with neural networks. We treat the problem in two setups; a 2D reconstruction and a 2D dynamic reconstruction task, and employ 2D and 3D deep learning networks, respectively. Our method optimizes in both the image and k-space domains, allowing for high reconstruction fidelity. Although the target data is undersampled with a Cartesian equispaced scheme, we train our model using both Cartesian and simulated non-Cartesian undersampling schemes to enhance generalization of the model to unseen data. Furthermore, our model adopts a deep neural network to learn and refine the sensitivity maps of multi-coil k-space data. Lastly, our method is jointly trained on both, undersampled cine and multi-contrast data.
### Multi-domain improves out-of-distribution and data-limited scenarios for  medical image analysis
 - **Authors:** Ece Ozkan, Xavier Boix
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2310.06737
 - **Pdf link:** https://arxiv.org/pdf/2310.06737
 - **Abstract**
 Current machine learning methods for medical image analysis primarily focus on developing models tailored for their specific tasks, utilizing data within their target domain. These specialized models tend to be data-hungry and often exhibit limitations in generalizing to out-of-distribution samples. Recently, foundation models have been proposed, which combine data from various domains and demonstrate excellent generalization capabilities. Building upon this, this work introduces the incorporation of diverse medical image domains, including different imaging modalities like X-ray, MRI, CT, and ultrasound images, as well as various viewpoints such as axial, coronal, and sagittal views. We refer to this approach as multi-domain model and compare its performance to that of specialized models. Our findings underscore the superior generalization capabilities of multi-domain models, particularly in scenarios characterized by limited data availability and out-of-distribution, frequently encountered in healthcare applications. The integration of diverse data allows multi-domain models to utilize shared information across domains, enhancing the overall outcomes significantly. To illustrate, for organ recognition, multi-domain model can enhance accuracy by up to 10% compared to conventional specialized models.
